Formula/schismtracker.rb

```diff
@@ -11,16 +12,11 @@ class Schismtracker < Formula
     sha256 "f892d60e3e74f352d7d9efbbf84b6470e4e751ffafe55c9b6ab9913bd611dba2" => :mavericks
   end
 
-  head do
-    url "http://schismtracker.org/hg/", :using => :hg
-
-    depends_on "autoconf" => :build
-  end
-
+  depends_on "autoconf" => :build
```

Comment:
Isn't the new source archive created from a source tag?

I'll revert this, but the build failed last time with the new source archive.